recipes/pdspy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+    name: pdspy
+    version: "1.5.0"
+
+source:
+    url: https://files.pythonhosted.org/packages/ee/b0/3e834fb25a597ef12a9764b92ee1c5cf6e1d4897805c9c1346ac0f70dfbf/pdspy-1.5.0.tar.gz
```

Comment:
please use the canonical form of the url as in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14